test/test_browser.py

```diff
@@ -4690,6 +4690,7 @@ def test_webgl_simple_extensions(self, simple_enable_extensions, webgl_version):
   @parameterized({
     'default': ([],),
     'closure': (['-sASSERTIONS', '--closure=1'],),
```

Comment:
The main run that we do of closure compiler always uses `ADVANCED_OPTIMIZATIONS` IIUC.

See https://github.com/emscripten-core/emscripten/blob/e0bd10a0b61e08d1ed30966b6d8351ea06376fd4/tools/link.py#L2218.

Here the closure_compiler() method is called without the `advanced` param, which means `advanced` defaults to `True`.

Reply:
Indeed if you build with `-O0` and `--closure=1` you can see (by adding -v) that it uses `--compilation_level ADVANCED_OPTIMIZATIONS`

Reply:
I am not sure how this works internally, but the output with `-O1` looks like the output with `SIMPLE` and the output with `-O2` looks like the output with `ADVANCED`. Maybe there's a more fine-grained option than SIMPLE vs ADVANCED that is causing this?

The externs seem to sometimes affect both builds, but more reliably affect the ADVANCED build.